src/Game/game.tsx

```diff
@@ -0,0 +1,185 @@
+/* eslint-disable no-param-reassign */
+import { throttle } from 'utils/throttle';
+import GameBG from '../img/game_sprites/city/main.png';
+import Hero from '../img/game_sprites/truck/main/body/main.png';
+import EnemyFirst from '../img/game_sprites/truck/enemys/main.png';
+import {
+    BG_POS,
+    HERO_POS,
+    CANVAS_X_SIZE,
+    CANVAS_Y_SIZE,
+    KEYS,
+} from './game_constants';
+
+const posX = 244;
+
+let enemPosX = CANVAS_X_SIZE + 20;
+let enemPosY = 220;
+let moveOn = 0;
+let moveOnEnemy = 0;
+
+const throttleEnemyMove = throttle(() => {
+    if (moveOnEnemy >= 732) {
+        moveOnEnemy = 0;
+    }
+    moveOnEnemy += posX;
+}, 80, null);
+
+function generateRandom(min = 0, max = 1): number {
```

Comment:
```suggestion
function generateRandom(): number {
	let rand = Math.random();
	rand = Math.floor(rand * 2);
	return rand;
}
```

Меньше расчетов, результат тот же